examples/api/lib/material/menu_bar/menu_bar.0.dart

```diff
@@ -0,0 +1,170 @@
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for MenuBar
+
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() => runApp(const SampleApp());
+
+enum MenuSelection {
+  about('About'),
+  showMessage('ShowMessage'),
+  colorMenu('Color Menu'),
+  colorRed('Red'),
+  colorGreen('Green'),
+  colorBlue('Blue'),
+  quit('Quit');
+
+  const MenuSelection(this.label);
+  final String label;
+}
+
+class SampleApp extends StatelessWidget {
+  const SampleApp({super.key});
+
+  static const String _title = 'MenuBar Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      title: _title,
+      home: Scaffold(body: MyMenuBarApp()),
+    );
+  }
+}
+
+class MyMenuBarApp extends StatefulWidget {
+  const MyMenuBarApp({super.key});
+
+  @override
+  State<MyMenuBarApp> createState() => _MyMenuBarAppState();
+}
+
+class _MyMenuBarAppState extends State<MyMenuBarApp> {
+  bool get showMessage => _showMessage;
+  bool _showMessage = false;
+  set showMessage(bool value) {
+    if (_showMessage != value) {
+      setState(() {
+        _showMessage = value;
+      });
+    }
+  }
+
+  Color get backgroundColor => _backgroundColor;
+  Color _backgroundColor = Colors.red;
+  set backgroundColor(Color value) {
+    if (_backgroundColor != value) {
+      setState(() {
+        _backgroundColor = value;
+      });
+    }
+  }
+
+  void _activate(MenuSelection selection) {
+    switch (selection) {
+      case MenuSelection.about:
+        showAboutDialog(
+          context: context,
+          applicationName: 'MenuBar Test',
+          applicationVersion: '1.0.0',
+        );
+        break;
+      case MenuSelection.showMessage:
+        showMessage = !showMessage;
+        break;
+      case MenuSelection.quit:
+        exit(0);
+      case MenuSelection.colorMenu:
+        break;
+      case MenuSelection.colorRed:
+        backgroundColor = Colors.red;
+        break;
+      case MenuSelection.colorGreen:
+        backgroundColor = Colors.green;
+        break;
+      case MenuSelection.colorBlue:
+        backgroundColor = Colors.blue;
+        break;
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        MenuBar(
+          menus: <PlatformMenu>[
+            MenuBarMenu(
+              autofocus: true,
+              label: 'Test App',
+              menus: <MenuItem>[
+                MenuBarItem(
+                  label: MenuSelection.about.label,
+                  onSelected: () => _activate(MenuSelection.about),
+                ),
+                MenuBarItem(
+                  // Add a builder so that that call to MenuBarController.of will be using
+                  // the correct context.
```

Comment:
Which `MenuBarController.of` is that? (there is no obvious one in the code sample blelow)